webpack/scenes/ContentViews/Table/ContentViewsTable.js

```diff
@@ -9,30 +9,73 @@ import tableDataGenerator from './tableDataGenerator';
 import getContentViews from '../ContentViewsActions';
 import CreateContentViewModal from '../Create/CreateContentViewModal';
 import CopyContentViewModal from '../Copy/CopyContentViewModal';
+import PublishContentViewWizard from '../Publish/PublishContentViewWizard';
+import { selectContentViews, selectContentViewStatus, selectContentViewError } from '../ContentViewSelectors';
 
-const ContentViewTable = ({ response, status, error }) => {
+const ContentViewTable = () => {
+  const response = useSelector(selectContentViews);
+  const status = useSelector(selectContentViewStatus);
+  const error = useSelector(selectContentViewError);
   const [table, setTable] = useState({ rows: [], columns: [] });
   const [rowMappingIds, setRowMappingIds] = useState([]);
   const [searchQuery, updateSearchQuery] = useState('');
-  const { results, ...metadata } = response;
   const loadingResponse = status === STATUS.PENDING;
   const [isModalOpen, setIsModalOpen] = useState(false);
   const [copy, setCopy] = useState(false);
+  const [cvResults, setCvResults] = useState([]);
+  const [metadata, setMetadata] = useState({});
+  const [cvTableStatus, setCvTableStatus] = useState(STATUS.PENDING);
+  const [isPublishModalOpen, setIsPublishModalOpen] = useState(false);
+  const [actionableCvDetails, setActionableCvDetails] = useState({});
   const [actionableCvId, setActionableCvId] = useState('');
   const [actionableCvName, setActionableCvName] = useState('');
-  function openForm() {
+  const [currentStep, setCurrentStep] = useState(1);
+
+  const openForm = () => {
     setIsModalOpen(true);
-  }
+  };
+
+  const openPublishModal = (rowInfo) => {
+    setActionableCvDetails({
+      id: rowInfo.cvId.toString(),
+      name: rowInfo.cvName,
+      composite: rowInfo.cvComposite,
+      version_count: rowInfo.cvVersionCount,
+      next_version: rowInfo.cvNextVersion,
+    });
+    setIsPublishModalOpen(true);
+  };
+
+  // Prevents flash of "No Content" before rows are loaded
```

Comment:
Does it, though? :thinking:

Reply:
Ya, but this is only on the main index page. Other tables still have this issue.